test/fixtures/v2/documentPolValidWithToken.ts

```
@@ -0,0 +1,48 @@
+}
+
+// Wrapped OA v2 document referencing a token registry on Polygon mainnet (chain ID 137).
+// tokenRegistry: TitleEscrowFactory address on Polygon from @tradetrust-tt/token-registry-v5 constants.
+// identity proof location: example.tradetrust.io (DNS-TXT with netId=137).
+// DOCUMENT_INTEGRITY (hash check) passes offline; DOCUMENT_STATUS requires the token to be
+// minted on-chain at 0xF94f95014304dC45B097439765A4D321bbE165c7 on Polygon mainnet.
+export const documentPolValidWithToken: WrappedDocument<CustomDocument> = {
+  version: SchemaId.v2,
+  data: {
+    version:
+      "eec1ca24-ed2b-4602-b133-5d0e6cb2096a:string:https://schema.openattestation.com/2.0/schema.json",
```

ESLint

[error] Delete `⏎·····`

(prettier/prettier)